my-rill-tutorial/1. Rill Basics/dashboards/commits___explore.yaml

```diff
@@ -4,7 +4,7 @@
 type: explore
 
 title: "Rill Basics Dashboard"
-metrics_view: commits___model_metrics
+metrics_view: commits___metrics
```

Comment:
The double underscore looks funky (since we don't use it elsewhere). I think let's stick with single underscores everywhere for consistency